### ISSUE TITLE
Cannot use template image + world file with geographic coordinates

### ISSUE THREAD

**Issue by dfgeorge** [OpenOrienteering/mapper#931]
### Steps to reproduce
1.  Map coordinate reference system set to UTM
2.  Template file has world file in WGS84 (degrees)
3.  Import into map as template, setting coordinate system to WGS84
template was generated using Terra Incognita (https://sourceforge.net/projects/terraincognita2/):
In settings set ArcView map: Units Lat/Lon then save as ArcView map. This produces a jpg with world file, as well as some ArcView files.

### Actual behavior
Scale of template is not correct when imported (approx 120 times too big).
Also tried setting EPSG to 4326 when importing, same result.
The file imports correctly into QGIS with EPSG set to 4326 , checked against a google satellite in QGIS.
Exporting the image from QGIS in UTM then importing into mapper works, but is a pain as file gets bigger and quality worst.

### Expected behavior
correctly positioned template.

### Configuration
Mapper Version: 0.7.0
Operating System: Windows 7, 32 bit

**Comment by dfgeorge** [comment]
I just exported the map to ocad and imported the same template using ocad, worked as expected.

**Comment by dg0yt** [comment]
Can you provide test files? You may replace the image contents with some plain color, or send me a direct e-mail.

**Comment by dg0yt** [comment]
(For attachments on Github, create a ZIP archive.)

**Comment by dg0yt** [comment]
IIRC you cannot use a different coordinate systems for the templates at the moment. And geographic coordinates which are not projected look like an extra complication. (Ground size of 1° of longitude varies strongly with latitude.) I think this case won't be handled before we fully switch to GDAL for geographic raster data.

On the other hand, transformation of a non-UTM image to UTM in Mapper will very likely have worse quality, too. Exporting as UTM short not generally result in much bigger file - this depends on resolution only.

**Comment by dfgeorge** [comment]
I have attached a zip file with boat a corrected and non corrected template in-case this gets looked at in the future. The corrected (UTM) template is slightly bigger as it was rotated for the different projection by QGIS.
The file got bigger as to create it I had to convert trough a geotif and didn't want to re-compress the jpg. The geotif was 10x the size of the original jpg.
QGIS is probably using gdal, so quality should be the same.
[bug931.zip](https://github.com/OpenOrienteering/mapper/files/1236926/bug931.zip)
